src/components/MediaSettings/VideoBackgroundEditor.vue

```diff
@@ -110,10 +140,111 @@ export default {
 		},
 	},
 
+	async mounted() {
+		// userRoot path
+		const userRoot = '/files/' + this.$store.getters.getUserId()
+
+		// Relative background folder path
+		const relativeBackgroundsFolderPath = this.$store.getters.getAttachmentFolder() + '/Backgrounds'
+
+		// Absolute backgrounds folder path
+		const absoluteBackgroundsFolderPath = userRoot + relativeBackgroundsFolderPath
```

Comment:
**nitpick:** These comments literally duplicates variable name up to a letter 😀
IMO, such comments are redundant and could disturb on refactoring.